sdk/servicebus/service-bus/CHANGELOG.md

```diff
@@ -13,9 +13,12 @@
 
 - `NamespaceProperties` interface property "messageSku" type changed from "string" to string literal type "Basic" | "Premium" | "Standard". [PR 11810](https://github.com/Azure/azure-sdk-for-js/pull/11810)
 
+- Internal improvement - For the operations depending on `$management` link such as peek or lock renewals, the listeners for the "sender_error" and "receiver_error" events were added to the link for each new request made before the link is initialized. This has been improved such that the listeners are reused.
```

Comment:
Might be worth mentioning the error message that would have popped up (for people searching).

Reply:
Mentioned